src/events/components/CalendarView/CalendarTile.tsx

```diff
@@ -20,15 +22,24 @@ export const createDayList = (amount: number, start: number): number[] => {
   return l;
 };
 
-export const CalendarEventTile = ({ events, active = true, day }: ITileProps) => {
+export const CalendarEventTile = ({ events, active = true, day, month }: ITileProps) => {
+  const thisDate = new Date();
+  const today = thisDate.getDate();
+  const thisMonth = thisDate.getMonth() + 1;
+  // tslint:disable-next-line: radix
+  const isToday = day === today && thisMonth === parseInt(month.toFormat('M')) ? true : false;

```

Comment:
When it comes to the ternary statement we can drop the `? true : false` part.

Example:
``` typescript
// In this case `isToday` would evaluate to a boolean, so adding the ternary would have no effect.
const isToday = day === today && thisMonth === parseInt(month.toFormat('M'));

// If we were to use it for something like a text to display, the ternary could be very useful
const dayText = day === today && thisMonth === parseInt(month.toFormat('M')) ? 'Today is the day' : 'Today is not the day'
```